samples/boards/stm32/power_mgmt/blinky/boards/b_u585i_iot02a.conf

```diff
@@ -0,0 +1,4 @@
+# Because the stm32U5 has a pre-div on the LPTIM clock source
+# the prescaler is twice the value of the stm32_lp_tick_source
+# SYS_CLOCK_TICKS_PER_SEC must be overwritten
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1024
```

Comment:
Shoudn't this be removed since the static prescaler has been removed in #65683 ?

Reply:
removed